plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/AllFilesTable.java

```diff
@@ -32,9 +32,9 @@ public AllFilesTable(SchemaTableName tableName, TypeManager typeManager, Table i
     @Override
     protected TableScan buildTableScan()
     {
-        Table filesTable = createMetadataTableInstance(getIcebergTable(), ALL_DATA_FILES);
+        Table allFilesTable = createMetadataTableInstance(getIcebergTable(), ALL_DATA_FILES);
```

Comment:
Squash this change with [Introduce $all_files Iceberg metadata table](https://github.com/trinodb/trino/pull/11206/commits/f13e7ab0eed6fb02d8be8e7132cb833ea2b2feed)